### ISSUE TITLE
[MAS1.4.3][Visual Requirements-Welcome] Luminosity Ratio is less than 4.5:1 for “Version 4” text 

### ISSUE THREAD

**Issue by awalia13** [microsoft/BotFramework-Emulator#1710]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 for “Version 4” text 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Version 4” text

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened 
3.Open 'Color Contrast Analyser' application.
4.Verify whether Luminosity Ratio is less than 4.5:1 for “Version 4” text or not 

**User Impact:**
Users with low vision or color blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
![MAS1 4 3_Welcome](https://user-images.githubusercontent.com/46575396/62769137-0576d980-bab6-11e9-8e5f-281b5287b3b9.png)

**Comment by tonyanziano** [comment]
@mewa1024 could you advise on what color to use in this scenario?

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator Version 4.5.2-78154 Version 4 and My bots section text is now 4.5:1
Issue got fixed hence we are closing this bug .
